packages/expo-image-manipulator/src/ExpoImageManipulator.web.ts

```diff
@@ -188,19 +188,10 @@ function getContext(canvas: HTMLCanvasElement): CanvasRenderingContext2D {
   return ctx;
 }
 
-function getResults(canvas: HTMLCanvasElement, options?: SaveOptions): ImageResult {
-  let base64;
-  if (options) {
-    const { format = 'png' } = options;
-    if (options.format === 'png' && options.compress !== undefined) {
-      console.warn('compress is not supported with png format.');
-    }
-    const quality = Math.min(1, Math.max(0, options.compress ?? 1));
-    base64 = canvas.toDataURL('image/' + format, quality);
-  } else {
-    // defaults to PNG with no loss
-    base64 = canvas.toDataURL();
-  }
+function getResults(canvas: HTMLCanvasElement, options: SaveOptions): ImageResult {
+  const { format = SaveFormat.JPEG, compress = 1 } = options;
+  const quality = Math.min(1, Math.max(0, compress));
+  const base64 = canvas.toDataURL('image/' + format, quality);
```

Comment:
base64 property (not uri) shouldn't include `data:image/xxx;base64,` and should only be present if requested

Reply:
Docs might be missing some aspect and I think it should be compared with the mobile platforms' behaviour and follow their behaviour. Our major goal is to provide unified API across all supported platforms (in some cases that might be unavailable, but in this case we can ensure it works the same everywhere I believe).